database/src/seeds/02_security_transform_eml_dwc_occurrences.ts

```diff
@@ -14,31 +14,87 @@ export async function seed(knex: Knex): Promise<void> {
   await knex.raw(`
     SET SCHEMA '${DB_SCHEMA}';
     SET SEARCH_PATH = ${DB_SCHEMA}, ${DB_SCHEMA_DAPI_V1};
-    INSERT INTO
-      security_transform
-      (persecution_or_harm_id, name, description,  transform)
-    VALUES
-      (1, 'DwC Occurrences', 'Assigns Persecution and Harm Rules',
-      ($transform$
-        WITH with_spatial_component AS
-          (SELECT spatial_component,
-          submission_spatial_component_id from submission_spatial_component where submission_observation_id = ? and jsonb_path_exists(spatial_component,'$.features[*] \\? (@.properties.type == "Occurrence")'))
-          SELECT jsonb_build_object(
-            'submission_spatial_component_id',
-            wsc.submission_spatial_component_id,
-            'spatial_data',
-              CASE
-                WHEN
-                  json_build_array(Lower(wsc.spatial_component->'features'->0->'properties'->'dwc'->>'associatedTaxa'))::jsonb
-                  <@ json_build_array('mountain goat', 'bighorn sheep', 'thinhorn sheep' , 'spotted owl', 'm-oram', 'm-ovca', 'm-ovca-ca', 'm-ovda', 'm-ovda-da', 'm-ovda-st', 'b-spow', 'b-spow-ca')::jsonb
-                THEN
-                    json_build_object()
-                ELSE
-                  NULL
-                END
-            ) spatial_component
-          FROM with_spatial_component wsc;
-        $transform$)
-      );
   `);
+
+  const response = await knex.raw(`
+    ${checkTransformExists()}
+  `);
+
+  if (!response?.rows?.[0]) {
+    await knex.raw(`
+      ${insertSecurityTransform()}
+    `);
+  } else {
+    await knex.raw(`
+    ${updateSecurityTransform()}
+  `);
+  }
 }
+
+const checkTransformExists = () => `
+  SELECT
+    security_transform_id
+  FROM
+    security_transform
+  WHERE
+    name = 'DwC Occurrences';
+`;
+
+/**
+ * SQL to insert DWC security transform
+ *
+ */
+const insertSecurityTransform = () => `
+  INSERT into security_transform
+    (name, description, record_effective_date, transform)
```

Comment:
This insert query is throwing some errors.

The table doesn't have a `record_effective_date` column. And it requires a `persecution_or_harm_id` to be provided.